tests/testapps/CGCatalog/CGCatalog/CGPathAddArcToPointViewController.h

```diff
@@ -0,0 +1,24 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#pragma once
+
+#import "CGCBaseViewController.h"
+#import <Foundation/Foundation.h>
+#import "CGSamplePage.h"
+
+@interface CGPathAddArcToPointViewController : CGCBaseViewController <CGSamplePage>

```

Comment:
When you merge up, would you mind adding these to CoreGraphics.Drawing tests, in the Path case? Or, file an issue and do it later. :)